compile.sh

```diff
@@ -1,57 +1,10 @@
 #!/bin/bash
-# compile all missing and outdated java classes, and copy .properties files over.
-# requires $HOME, $TARGET, and $CLASSPATH to be set, or maven to be installed.
-# do not call directly, indirectly invoked via ./run.sh
-
-# store the path of the conll-rdf directory in HOME
-HOME=$( dirname -- "$(realpath -- "$0")");
-TARGET="$HOME/target/classes";
-CLASSPATH="$TARGET:$HOME/lib/*";
-
-mkdir -p "$TARGET";
-
-JAVAS="";
-# Find all java classes that need to be compiled
-for java in $(find $HOME/src/main/java -name "*.java"); do
-	class=$TARGET${java#$HOME/src/main/java}
-	class=${class%.java}.class
-	if [ ! -e $class ] || [ $java -nt $class ] || [ " "$1 = " -f" ]; then
-		JAVAS="$JAVAS $java"
-	fi;
-done;
-
-# NEW: build with Maven
-if echo $JAVAS | egrep '.' >/dev/null; then
```

Comment:
I'm so happy to throw all of this out :D
`./compile.sh` is no longer called by run.sh, as I included the compile goal in run.sh. Maven is better at, and just as fast to check if anything needs compiling or copying over. And as we're already calling maven there ... it's free